cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -281,16 +281,31 @@ def test_assay_ontology_term_id(self):
 
     def test_cell_type_ontology_term_id(self):
         """
-        cell_type_ontology_term_id categorical with str categories. This MUST be a CL term.
+        cell_type_ontology_term_id categorical with str categories. This MUST be a CL term, and must NOT match forbidden
+        columns defined in schema
         """
 
         # Not a valid term
-        self.validator.adata.obs.loc[self.validator.adata.obs.index[0], "cell_type_ontology_term_id"] = "EFO:0000001"
-        self.validator.validate_adata()
-        self.assertEqual(
-            self.validator.errors,
-            ["ERROR: 'EFO:0000001' in 'cell_type_ontology_term_id' is not a valid " "ontology term id of 'CL'."],
-        )
+        with self.subTest(forbidden_term="EFO:0000001"):
+            self.validator.adata.obs.loc[
+                self.validator.adata.obs.index[0], "cell_type_ontology_term_id"
```

Comment:
What is the benefit of using the numpy method of slicing `foo[ x, y]`, as opposed to pythons way using `foo[x][y]`?

Reply:
this is the syntax for the [loc](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.loc.html) accessor function for pandas dataframes